components/function-controller/internal/controllers/serverless/build_resources_test.go

```diff
@@ -543,6 +543,11 @@ func TestFunctionReconciler_buildJob(t *testing.T) {
 					Spec: serverlessv1alpha2.FunctionSpec{
 						Runtime: testCase.Runtime,
 						Source:  serverlessv1alpha2.Source{Inline: &serverlessv1alpha2.InlineSource{}},
+						ResourceConfiguration: &serverlessv1alpha2.ResourceConfiguration{
```

Comment:
Do we need this, as it's optional and wasn't in previous tests

Reply:
Reverted